app-information.html

```diff
@@ -0,0 +1,369 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>
+      Web App Manifest - App Information
```

Comment:
Should we write it out "Application Information"?

Reply:
That’s fine with me